documentation/config_logging_advanced.md

```diff
@@ -105,6 +106,7 @@ To add JSON logging for Puppet Server's HTTP requests, modify the `request-loggi
             </providers>
         </encoder>
     </appender>
+{% endraw %}
 ```
```

Comment:
We only have to do it around the example that includes `{%` in the code example, and that's the only one I see.